data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/deserializers/SimpleCracDeserializer.java

```diff
@@ -48,7 +48,14 @@ public SimpleCrac deserialize(JsonParser jsonParser, DeserializationContext dese
         }
         String name = jsonParser.nextTextValue();
 
-        SimpleCrac simpleCrac = new SimpleCrac(id, name);
+        jsonParser.nextToken();
+        if (!jsonParser.getCurrentName().equals(INSTANTS)) {
+            throw new FaraoException("'instants' field is expected in the fifth line of the json file.");
+        }
+        jsonParser.nextToken();
+        Set<Instant> instants = jsonParser.readValueAs(new TypeReference<Set<Instant>>() { });
+
```

Comment:
Same here. Is it really necessary to define the instants in the JSON ? As they already are defined by the enum ?